.gitignore

```diff
@@ -42,3 +42,7 @@ cypress/videos/**
 public/env-config.js
 /junit.xml
 /oclapi/
+
+# MAC OS
+.DS_Store
+openmrs-ocl-client.iml
```

Comment:
If we're going to add an exclusion for IntelliJ artefacts here, it should be `*.iml` and probably placed near the `.idea/*` exclusion, since `iml` files are unique to macOS.